src/Money.php

```diff
@@ -36,6 +38,11 @@
     private static $calculator;
 
     /**
+     * @var Currencies
+     */
+    private static $currencies;
```

Comment:
I am not really sure that this is a good idea. But I don't see a better option. Alternative: remove conversion from the Money object and move it to a Converter object.

/cc @frederikbosch